src/resources/address.js

```diff
@@ -8,8 +9,8 @@ export const propTypes = Object.assign({}, baseAddressPropTypes, {
   carrier_facility: T.string,
   federal_tax_id: T.string,
   state_tax_id: T.string,
-  verify: T.array,
-  verify_strict: T.array,
+  verify: T.oneOfType([T.bool, T.string, T.array]),
```

Comment:
Thought: Same question as on Python, shouldn't we be guiding users to only using booleans, rather than allowing them to continue doing it "the bad way"?

Reply:
See my reply on the Python PR. Using arrays isn't "bad", there's just a more concise way via the Boolean.